proposal/comp110_architecture_proposal.tex

```diff
@@ -14,61 +14,74 @@
 
 \section*{Topic}
 
-My essay will be on
-% Uncomment as appropriate:
-%   convexity-based collision detection for a 2D game engine.
-%   procedural level generation for a 2D platform game.
+My essay will be on procedural level generation for a 2D platform game.
 
 \section*{Paper 1}
-% This is an example! Replace the details with a paper relevant to your chosen topic.
+
 \begin{description}
-\item[Title:] A Mathematical Theory of Communication
-\item[Citation:] \cite{shannon}
-\item[Abstract:] ``The recent development of various methods of modulation such as PCM and PPM which exchange bandwidth for signal-to-noise ratio has intensified the interest in a general theory of communication. A basis for such a theory is contained in the important papers of Nyquist and Hartley on this subject. In the present paper we will extend the theory to include a number of new factors, in particular the effect of noise in the channel, and the savings possible due to the statistical structure of the original message and due to the nature of the final destination of the information.''
-\item[Web link:] \url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6773024}
-\item[Full text link:] \url{http://ieeexplore.ieee.org/stamp/stamp.jsp?tp=&arnumber=6773024}
-\item[Comments:] This is a seminal article, which laid the foundations for what we now call information theory.
-	It has been cited more than $76\,000$ times, making it one of the most-cited articles in computer science.
+\item[Title:] Experience-Driven Procedural Content Generation
```

Comment:
An interesting paper, which will be a good source of supporting material. However it appears to be a survey of prior work rather than a primary source.